include/wil/Microsoft.Windows.Telemetry.Common.h

```diff
@@ -0,0 +1,136 @@
+#pragma once
+//! @file
+//! WIL Common Helpers: Provides broadly applicable, dependency-free pure C++ helpers, macros and type traits.
+
+#ifndef __MICROSOFT_WINDOWS_TELEMETRY_COMMON_INCLUDED
+#define __MICROSOFT_WINDOWS_TELEMETRY_COMMON_INCLUDED
+
+#include <wil/common.h>
+
+#if defined(__cplusplus) && !defined(__WIL_MIN_KERNEL)
+
+/// @cond
+#ifndef WIL_HIDE_DEPRECATED_1611

```

Comment:
Don't introduce things that are already deprecated; remove these. Seems like this entire header can be deleted.